### PR TITLE
Prepare the JSPs for β

### PR DESCRIPTION
**Pull request by eggrobin**
Update the documentation, allow for a Uβ prefix, and fix the logic that was meant to disallow an unqualified U16 prefix.

See it in staging: 

* [Unikemet query](https://unicode-jsps-staging-o2ookmn2oq-uc.a.run.app/UnicodeJsps/list-unicodeset.jsp?a=%5Cp%7BU%CE%B2%3AkEH_FVal%3D%2F.%2F%7D&g=U%CE%B2%3AkEH_FVal&i=U%CE%B2%3AkEH_Func%3BU%CE%B2%3AkEH_Desc)
* [All new characters](https://unicode-jsps-staging-o2ookmn2oq-uc.a.run.app/UnicodeJsps/list-unicodeset.jsp?a=%5CP%7BU16%CE%B2%3ACn%7D-%5CP%7BCn%7D&g=&i=) (compare [the α répertoire](https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5CP%7BU16%CE%B1%3ACn%7D-%5CP%7BCn%7D&g=&i=&showDevProperties=1) on the current live JSPs, with the two poofed shriis).

### CODE REVIEW

**Comment by markusicu** [comment]
FYI @josh-hadley @nedley @srl295

**Comment by markusicu** [comment]
I am surprised that it still reports using ICU 74. I thought Mihai had updated it to 75.

**Comment by eggrobin** [comment]
Hmm, the top-level POM even says
`        <icu.version>76.0.1-SNAPSHOT</icu.version>`

**Comment by eggrobin** [comment]
Maybe I missed a build step? I will take another look. I don’t think there is anything 74-related in the source tree, so I think this is operator error (which points to the brittleness of the build process but let’s not block ourselves on that).

File: unicodetools/src/main/java/org/unicode/text/UCD/VersionedProperty.java [hunk 1]
```diff
@@ -96,8 +98,15 @@ public VersionedProperty set(String xPropertyName) {
...
             } else {
                 version = names[0].substring(1);
-                if (versionAliases.containsValue(version)) {
-                    throw new IllegalArgumentException("Invalid version " + version);
+                if (versionAliases.containsValue(
+                        VersionInfo.getInstance(version).getVersionString(3, 3))) {
+                    throw new IllegalArgumentException(
+                            "Unreleased version "
+                                    + version
+                                    + "; use suffix: "
```
[the remark this answers is not in the document]
Comment (eggrobin) [reply unicodetools/src/main/java/org/unicode/text/UCD/VersionedProperty.java:106]: The version itself is a prefix on the property; but this is telling you to put a suffix on the version.

**Comment by eggrobin** [comment]
On second thought, if I managed to use 76 which we are meant to be using, I would be shipping ICU with α data. Let’s not go there.

**Review by markusicu — APPROVED** [review]

**Pull request merged**